src/context.cpp

```diff
@@ -871,20 +831,14 @@ namespace librealsense
             if( ! _md_enabled )
                 return;
 
-            auto video_iter = _stream_name_to_video_syncer.find( stream_name );
-            if( video_iter != _stream_name_to_video_syncer.end() )
-                video_iter->second.enqueue_metadata( std::move( dds_md ) );
+            auto it = _stream_name_to_syncer.find( stream_name );
+            if( it != _stream_name_to_syncer.end() )
+                it->second.enqueue_metadata(
+                    std::stoull( rsutils::json::get< std::string >( dds_md["header"], "frame-id" ) ),
```

Comment:
Might throw

Reply:
That's OK here, it'll be caught and report it